README.md

```
@@ -257,6 +257,22 @@ ulimit -n 10240
 cargo test -- --test-threads=$(nproc)
 ```
 
+## Run Integration Tests Only
+
+Integration tests are located in the `integration_tests/` directory and don't require a running oxen server:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)